### ISSUE TITLE
Does this project support next.js dApps?

### ISSUE THREAD

**Issue by DannyFung97** [NoahZinsmeister/web3-react#473]

**Comment by ubinatus** [comment]
Indeed. Noah has added a [Next.js example](https://web3-react-mu.vercel.app/). I recommend to check out the example source code, since v8 full documentation is not yet self explanatory. It will help you get an idea of how to use it for your Next.js dApp. 
https://github.com/NoahZinsmeister/web3-react/tree/main/packages/example-next

Since this v8 is in beta, you can still check out previous v6, which has been widely used by [various projects](https://github.com/NoahZinsmeister/web3-react/tree/v6#projects-using-web3-react).
https://github.com/NoahZinsmeister/web3-react/tree/v6

**Comment by ubinatus** [comment]
Indeed. Noah has added a [Next.js example](https://web3-react-mu.vercel.app/). I recommend to check out the example source code, since v8 full documentation is not yet self explanatory. It will help you get an idea of how to use it for your Next.js dApp. 
https://github.com/NoahZinsmeister/web3-react/tree/main/packages/example-next

Since this v8 is in beta, you can still check out previous v6, which has been widely used by [multiple projects](https://github.com/NoahZinsmeister/web3-react/tree/v6#projects-using-web3-react).
https://github.com/NoahZinsmeister/web3-react/tree/v6

**Comment by agcty** [comment]
Yes, it does support Next.js! There are a few usability issues that are currently being worked on but these should be resolved soon. Personally want to work on better docs after that.

**Comment by mattcasey** [comment]
I'm using v6 fine with next.js. Eager to move to v8 but waiting for an 'official' release I guess.

**Comment by agcty** [comment]
@mattcasey even though v8 is still in beta, I feel like it is already more stable than v6, you can check out the ProviderExample https://github.com/NoahZinsmeister/web3-react/blob/main/packages/example-next/components/ProviderExample.tsx#L33 to see how you can implement it, docs will be available soon!

**Comment by NoahZinsmeister** [comment]
closing, working on docs as part of #353!
